deploy/cloudformation/iiif-service.yml

```diff
@@ -255,6 +265,42 @@ Resources:
     Type: AWS::S3::Bucket
     Description: This is the target bucket that the IIIF service will write to when it generates its cache of derivative.
 
+  DebugPermissions:
+    Type: AWS::IAM::ManagedPolicy
+    Properties:
+      Roles:
+        - !ImportValue
+            Fn::Sub: ${InfrastructureStackName}:DebugRoleName
+      PolicyDocument:
+        Version: '2012-10-17'
+        Statement:
+          - Sid: AllowReadBuckets
+            Effect: Allow
+            Action:
+              - 's3:GetObject'
+              - 's3:ListBucket'
+            Resource:
+              - !GetAtt ImageDerivativeCacheBucket.Arn
+              - !Sub ${ImageDerivativeCacheBucket.Arn}/*
+
+  DataAdminPermissions:
+    Type: AWS::IAM::ManagedPolicy
+    Properties:
+      Roles:
+        - !ImportValue
+            Fn::Sub: ${InfrastructureStackName}:DataAdminRoleName
+      PolicyDocument:
+        Version: '2012-10-17'
+        Statement:
+          - Sid: AllowReadBuckets
+            Effect: Allow
+            Action:
```

Comment:
Would there ever be a need for the `DataAdmin` to edit items in the cache?

Reply:
That's a good question. I would think the only time you would need to act on the cache would be to delete objects when there is an issue. I'm not sure if this can be done through the Cantaloupe admin interface though. @jonhartzler might have a stronger opinion on this.

Reply:
My vote is to not allow this yet, and add the permissions if we find the need.